frontend/src/404/404.js

```diff
@@ -1,26 +1,45 @@
-import React, {Component} from 'react';
-import './404.css';
-import {Box} from '@material-ui/core';
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  box404: {
+    backgroundColor: 'beige',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '100vh',
+    fontSize: '900%',
+    textAlign: 'center',
+  },
+  text404: {
+    margin: '0px',
+  },
+  subtext: {
+    marginTop: '0px',
+    fontSize: '20%',
+  },
+  homelink: {
+    color: '',
+    textDecoration: 'none',
+  },
+});
 
 /**
- * Represents the 404 Page for CloudHaven
+ * asdfas
+ *
+ * @export
+ * @return {*}
  */
```

Comment:
It might be good to leave a blurb about the page here and have a more descriptive return type. It's obvious for the 404 page but would fit with the JSDoc on most of the other components.